meshtastic/interfaces/ble/compatibility_service.py

```
@@ -0,0 +1,319 @@
+        if callable(legacy_safe_execute) and not _is_unconfigured_mock_callable(
+            legacy_safe_execute
+        ):
+            return cast(Callable[..., Any], legacy_safe_execute)
+        return None
+
+    @staticmethod
+    def _safe_execute(
+        iface: "BLEInterface",
+        func: Callable[[], Any],
+        *,
+        default_return: Any = None,
```

Ruff (0.15.5)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `default_return`

(ANN401)